dotCMS/src/main/java/com/dotmarketing/filters/InterceptorFilter.java

```diff
@@ -40,7 +40,7 @@ private void addInterceptors(final FilterConfig config) {
         delegate.add(new ResponseMetaDataWebInterceptor());
         delegate.add(new EventLogWebInterceptor());
         delegate.add(new CurrentVariantWebInterceptor());
-        delegate.add(new AnalyticsTrackWebInterceptor());
+        //delegate.add(new AnalyticsTrackWebInterceptor()); // turn on when needed.
```

Comment:
Shouldn't be a FF better to be able to turn on without code changes required?

Reply:
this is just something temporal